vsim/Makefile

```diff
@@ -32,3 +32,13 @@ clean:
 	rm -rf $(junk) simv* csrc *.key DVE* *.h *.a *.daidir $(generated_dir)
 
 .PHONY: default all debug clean
+
+sim_vsrcs = $(base_dir)/vsrc/$(TB).v $(generated_verilog) $(verilog)
+
+sim_csrcs = $(generated_cppfiles)              \
+			$(base_dir)/csrc/remote_bitbang.cc \
+			$(base_dir)/csrc/SimDTM.cc \
+			$(base_dir)/csrc/SimJTAG.cc
```

Comment:
Are these tabs?

Reply:
They look to be. I suppose they shouldn't be, right?

Reply:
I would do spaces and line up the `$` but it's not that important

Reply:
The deleted Makefrag seems to have mixed spaces and tabs for this